### PR TITLE
EXP-557 fix(receipt): now showing url for principal receipt

### PR DESCRIPTION
**Pull request by allangrds**
## Contexto
Não mostrando o campo de URL quando acessada as informações do recebedor principal.

## Como testar
- Pegue um id de um recebedor principal e o id de um recebedor que não é principal e execute o código abaixo no seu terminal, substituindo a api_key e id do recebedor:
```
curl -L -X PUT 'https://api.pagar.me/1/recipients/<id do recebedor default>' \
-H 'content-type: application/json' \
--data-raw '{
    "api_key": "<api key>",
    "register_information": {
        "site_url": "https://google.com"
    }
}'
```
- Acessar o recebedor principal e não deve ser vista url na aba "mais informações"
- Acessar outro recebedor com a url, e ver a url

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8e345051fee5a717f2c469a571a6932caed85549
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  

  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8e345051fee5a717f2c469a571a6932caed85549
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  

  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8e345051fee5a717f2c469a571a6932caed85549
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!  

  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: e084bfa2267c53fd1eba372061ff8c0dccb22be4
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: e084bfa2267c53fd1eba372061ff8c0dccb22be4
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: e084bfa2267c53fd1eba372061ff8c0dccb22be4
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: e084bfa2267c53fd1eba372061ff8c0dccb22be4
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 15d2beaf72313c400fa94f41ad097dba6b397922
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 15d2beaf72313c400fa94f41ad097dba6b397922
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 15d2beaf72313c400fa94f41ad097dba6b397922
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 15d2beaf72313c400fa94f41ad097dba6b397922
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/pilot/src/pages/Recipients/Detail/index.js [hunk 1]
```diff
@@ -48,7 +49,12 @@ const mapStateToProps = (state = {}) => {
     },
```
Comment (vagnervst) [inline packages/pilot/src/pages/Recipients/Detail/index.js:49]: Ao fazer o login nós guardamos o recebedor principal no redux, pra facilitar pode puxar de lá (a vantagem é não precisar lidar com environment)

```suggestion
    },
    defaultRecipient: {
      id: defaultRecipientId,
    }
```
Assim basta remover o `default_recipient_id` de company

File: packages/pilot/src/containers/AddRecipient/ConfirmStep/ReceiverInfo/index.js [hunk 2]
```diff
@@ -30,16 +30,22 @@ const renderReceiverNameEmailInfo = (identification, t) => {
...
 
-const renderReceiverUrlPhoneInfo = (identification, t) => {
+const renderReceiverUrlPhoneInfo = (
+  identification, t, hash, defaultRecipientId
+) => {
   const url = t('pages.add_recipient.url')
   const phone = t('pages.add_recipient.phone')
   const urlType = identification[`${identification.documentType}Url`]
   const phoneType = identification[`${identification.documentType}Phone`]
+  const isPrimaryReceipt = hash === defaultRecipientId[
```
Comment (vagnervst) [inline packages/pilot/src/containers/AddRecipient/ConfirmStep/ReceiverInfo/index.js:40]: Só um typo aqui

```suggestion
  const isPrimaryRecipient = hash === defaultRecipientId[
```

**Review by vagnervst — COMMENTED** [review]
Só estes dois pontos que tive dificuldades pra testar por causa da environment

**Review by vagnervst — APPROVED** [review]

**Review by diegodsgarcia — COMMENTED** [review]
Percebi um pequeno bug quando o recebedor não tem numero declarado:
image

Acho que isso pode ser resolvido no cockpit
```diff
diff --git a/packages/cockpit/src/recipient/detail/format/index.js b/packages/cockpit/src/recipient/detail/format/index.js
index 33e4167b..1e352b28 100644
--- a/packages/cockpit/src/recipient/detail/format/index.js
+++ b/packages/cockpit/src/recipient/detail/format/index.js
@@ -39,7 +39,7 @@ function formatAntecipationAndTransferConfiguration (data) {
   const { ddd, number } = pathOr({}, ['phone_numbers', 0], register)
 
   if (register !== null) {
-    phoneNumber = `(${ddd}) ${number}`
+    phoneNumber = (ddd && number) ? `(${ddd}) ${number}` : ''
   }
 
   if (data.bank_account.document_type === 'cpf') {

```

**Review by diegodsgarcia — APPROVED** [review]

**Pull request merged**